modules/statics/src/base.ts

```diff
@@ -204,6 +204,11 @@ export enum CoinFeature {
    * This coin supports cold wallets that use a TSS signing protocol
    */
   TSS_COLD = 'tss-cold',
+
+  /**
+   * This coin uses sha256 hash function for ECDSA TSS signatures
+   */
+  USES_SHA256_WITH_ECDSA_TSS = 'uses-sha256-with-ecdsa-tss',
```

Comment:
```suggestion
  SHA256_WITH_ECDSA_TSS = 'uses-sha256-with-ecdsa-tss',
```
Nit: Can we remove the USES?

Reply:
done